packages/gensx-core/src/utils/project-config.ts

```diff
@@ -1,13 +1,15 @@
 import { readFileSync } from "node:fs";
 import path from "node:path";
 
+import { load } from "js-yaml";
 import { z } from "zod";
 
 // Define schema for gensx.yaml
 const ProjectConfigSchema = z.object({
   projectName: z.string(),
   environmentName: z.string().optional(),
   description: z.string().optional(),
+  public: z.boolean().optional(),
```

Comment:
seems like there's a mismatch between here and the schema we use in the gensx package? `public` vs `publicWorkflows`